### PR TITLE
Ensure Fleet REST yaml tests work in serverless

### PR DESCRIPTION
**Pull request by kingherc**
Use new gradle plugin for the yaml tests so they can be executed in serverless CI as well.

Relates ES-8275

### CODE REVIEW

File: x-pack/plugin/fleet/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/fleet/20_wait_for_checkpoints.yml [hunk 1]
```diff
@@ -105,6 +105,7 @@ setup:
         index: "test-after-refresh"
         allow_partial_search_results: false
         wait_for_checkpoints: 2
+        wait_for_checkpoints_timeout: 1m
```
Comment (kingherc) [inline x-pack/plugin/fleet/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/fleet/20_wait_for_checkpoints.yml:108]: A timetous >30sec is needed in order to bypass the condition set by https://github.com/elastic/elasticsearch/pull/96145 and make the "wait for unissued seqno" error to appear. Here and in the below cases.

**Comment by kingherc** [comment]
CI failure seems to have started popping up today in general (see [here](https://es-delivery-stats.elastic.dev/app/r/s/ToaqZ)) so I expect it'll be muted soon. Please ignore it.

**Comment by kingherc** [comment]
@elasticmachine update branch

**Comment by kingherc** [comment]
CI is a bit unstable, probably because of the automatic unmute of some tests. Expecting to rebase soon after they are again muted, to have a green CI. Feel free to review in the meantime.

**Review by henningandersen — APPROVED** [review]
LGTM.

**Pull request merged**